tapir/wirgarten/forms/registration/bestellcoop.py

```diff
@@ -0,0 +1,69 @@
+    <li>Hülsenfrüchte (Linsen, Bohnen, Erbsen etc. in 2-5kg-Packungen)</li>
+    <li>Öle & Essig (3-5l-Kanistern)</li>
+    <li>Konserven (Passata, Brotaufstriche - jeweils im Set von mehreren Gläsern)</li>
+    <li>Saaten & Kerne (0,5-2kg)</li>
+    <li>Nüsse (1-5kg)</li>
+    <li>Tee & Kaffee vom avenir (0,5-1kg)</li>
+</ul>
+<p>Die Produkte beziehen wir aktuell zum einen von lokalen Produzent:innen wie dem a-venir, HeyHo oder Meyers Windmühle. Ergänzt wird das Angebot durch verschiedene Bio-Großhändler. Wir freuen uns über deine Anregungen zur Erweiterung des Angebots und der regionalen Produzent:innen, denn wir wollen die Wertschöpfung und das Lebensmittelhandwerk vor Ort weiter stärken.</p>
+</div>
+<div style="margin-left:2em"><img style="max-height:100%" src="/static/wirgarten/images/bestellcoop.webp"/></div>
+</div>
+<hr/>
+<h5>Wie funktioniert die BestellCoop?</h5>
+<ul>
+    <li>Deine Mitgliedschaft läuft je für sechs Monate</li>
+    <li>Du kannst (musst aber nicht) alle zwei Monate bestellen</li>
+    <li>Du bestellst bequem über den WirGarten-Webshop</li>
+    <li>Du bekommst deine Bestellung zu dir nach Hause geliefert</li>
+    <li>Du erhältst alle wichtigsten Informationen gesammelt per Newsletter rechtzeitig vor Beginn der nächsten Bestellrunde</li>
+</ul>
+<strong>Das Angebot gilt nur für Mitglieder der WirGarten Lüneburg eG.</strong>
+<hr/>
+<h5>Was kostet die Mitgliedschaft in der BestellCoop?</h5>
+<p>Wir bieten die BestellCoop kostendeckend an. Es gibt keine Preisaufschläge und wir reichen die Großhandelspreise eins zu eins an dich weiter. Gleichzeitig sollen keine Belastungen für unser Kerngeschäft - den Gemüsebau - entstehen. Zur Deckung der administrativen Aufwendungen haben wir folgendes Preismodell entwickelt:</p>
+<ul>
+    <li>Du zahlst insgesamt <b>18€ Mitgliedsgebühr</b> für die gesamten sechs Monate - das entspricht <b>3€ pro Monat</b></li>
+    <li>Sofern du bestellst, kommen nochmals <b>7€ Bestellgebühr</b> oben drauf</li>
+</ul>
+<p>Du zahlst also maximal 39 €, wenn du in allen drei Bestellrunden der sechsmonatigen Mitgliedschaft bestellst. Über die Großhandelspreise hast du im Schnitt bis zu 30-40% Ersparnis zu den Einzelhandelspreisen. Die Mitgliedschaft in der BestellCoop kann sich also schon ab einem Bestellwert von insgesamt ca. 120€ in den sechs Monaten lohnen.</p>
+"""
+    summary = """<span style="float:right; font-size:1.25rem">= <strong><span id="bestellcoop_total">0.00</span> €</strong> / Monat<script>
+        const resultElem = document.getElementById('bestellcoop_total');
+        const checkbox = document.getElementById('id_BestellCoop-bestellcoop');
+
+        const handleChange = () => {{
+            resultElem.innerText = (checkbox.checked ? {price}: 0).toFixed(2)
+        }}
+
+        checkbox.addEventListener('change', handleChange)
+
+        handleChange()
+    </script>
+    """.format(
+        price=BESTELLCOOP_PRICE
+    )
```

Comment:
Is there no way to put this in a template file?

Reply:
The idea in the beginning was to have the possibility for the admin to change/customize these texts.
BUT the texts got very long and complex, so I doubt that this will ever happen. Yes I can put in in template files (or better snippets that I include in the wizard template).